azure/azure-release-template.yml

```diff
@@ -65,62 +69,74 @@ extends:
           export PACT_PROVIDER_URL=https://$APIGEE_ENVIRONMENT.api.service.nhs.uk/$SERVICE_BASE_PATH
 
           if [[ "$APIGEE_ENVIRONMENT" == *"sandbox"* ]]; then
-            export PACT_PROVIDER=nhsd-apim-eps-sandbox
+          export PACT_PROVIDER=nhsd-apim-eps-sandbox
```

Comment:
We've got a weird mix of indenting here. E.g. line 72 isn't indented but line 77 is.

Reply:
I think the indenting makes it easier to keep track of what's going on so my preference would be to keep it